docroot/modules/custom/sitenow_ccom/sitenow_ccom.module

```diff
@@ -0,0 +1,36 @@
+<?php
+
+/**
+ * @file
+ * Contains lb_enhancements.module.
+ */
+
+use Drupal\Component\Utility\Html;
+use Drupal\Component\Utility\NestedArray;
+use Drupal\Core\Cache\Cache;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\Core\Render\Element;
+use Drupal\Core\StringTranslation\TranslatableMarkup;
+use Drupal\Core\Url;
+use Drupal\ctools_views\Plugin\Display\Block;
+use Drupal\layout_builder\Form\ConfigureBlockFormBase;
+use Drupal\layout_builder_custom\LayoutBuilderPreRender;
+use Drupal\layout_builder_custom\LayoutBuilderStylesHelper;
+use Drupal\layout_builder_custom\Plugin\Display\ListBlock;
+use Drupal\layout_builder_styles\LayoutBuilderStyleInterface;
+use Drupal\uiowa_core\Element\Card;
+use Drupal\uiowa_core\HeadlineHelper;
+use Drupal\views\ViewExecutable;
+
+/**
+ * Implements hook_preprocess_HOOK().
+ */
+function sitenow_ccom_preprocess_block(&$variables) {
+  if (isset($variables['attributes']['id']) && $variables['attributes']['id'] ==='block-uids-base-footercontactinfo') {
+    $current_path = \Drupal::service('path.current')->getPath();
```

Comment:
Should we use `\Drupal::request()->getRequestUri();` to get the alias path if it exists? Similar to our [login redirect](https://github.com/uiowa/uiowa/blob/main/docroot/modules/custom/uiowa_core/uiowa_core.module#L917)